src/cdk/overlay/_index.scss

```diff
@@ -96,13 +96,8 @@ $backdrop-animation-timing-function: cubic-bezier(0.25, 0.8, 0.25, 1) !default;
   }
 
   .cdk-overlay-transparent-backdrop {
-    // Note: as of Firefox 57, having the backdrop be `background: none` will prevent it from
-    // capturing the user's mouse scroll events. Since we also can't use something like
-    // `rgba(0, 0, 0, 0)`, we work around the inconsistency by not setting the background at
-    // all and using `opacity` to make the element transparent.
-    &, &.cdk-overlay-backdrop-showing {
-      opacity: 0;
```

Comment:
why was the `opacity: 0` removed? this seems to be breaking screenshots in some apps

Reply:
It's been a while since I opened the PR, but I _think_ it was because the `opacity` would actually cause the transition to run, even though it's animation from transparent to transparent.

Reply:
I think there may be something not quite right about it. When I presubmitted this I wound up with some apps that had opaque backdrops covering everything